cluster-up/cluster/okd-4.1/provider.sh

```diff
@@ -11,7 +11,7 @@ function _port() {
 }
 
 function up() {
-    params="--random-ports --background --prefix $provider_prefix --master-cpu 5 --workers-cpu 5 --registry-volume $(_registry_volume) kubevirtci/${image}"
+    params="--random-ports --background --prefix $provider_prefix --master-cpu 6 --workers-cpu 6 --registry-volume $(_registry_volume) --workers 2 kubevirtci/${image}"
```

Comment:
now we have three nodes for okd cluster?

Reply:
yep I prefer to run migration tests on separate workers

Reply:
we need to update the amount of ram we need to start an okd cluster in the readme?

Reply:
Under the README I do not create OKD with two workers - https://github.com/kubevirt/kubevirtci/pull/126/files#diff-0fe36727ce1f5b7b5dddbc90866b45c3R60, so it should be ok

Reply:
you are right

Reply:
it does look like system requirements have jumped here though. Are there not 3 nodes involved now instead of 2?

Reply:
yep it 3 nodes, 2 workers and 1 master, but it should not affect our runs under the CI because we dedicate 29GiB of memory of the OKD environment. I think to have 2 workers nodes and avoiding to run the workload on the master(because it already overloaded with the infrastructure) can make our CI more stable(at least for OKD case).

Reply:
For example master under the OKD has warnings of the sort
```
  Warning  EvictionThresholdMet     5m53s               kubelet, test-1-d7xg4-master-0  Attempting to reclaim ephemeral-storage
  Normal   NodeHasDiskPressure      5m48s               kubelet, test-1-d7xg4-master-0  Node test-1-d7xg4-master-0 status is now: NodeHasDiskPressure
  Normal   NodeHasNoDiskPressure    44s (x51 over 24m)  kubelet, test-1-d7xg4-master-0  Node test-1-d7xg4-master-0 status is now: NodeHasNoDiskPressur
```
and it can break tests run

Reply:
> yep it 3 nodes, 2 workers and 1 master, but it should not affect our runs under the CI because we dedicate 29GiB of memory of the OKD environment. I think to have 2 workers nodes and avoiding to run the workload on the master(because it already overloaded with the infrastructure) can make our CI more stable(at least for OKD case).

I agree that this has the potential to make CI more stable. 

Since we have two worker nodes now. This is increasing the system requirements necessary to run the okd provider locally though, which is an important use case for us in our dev environments.

What are the system requirements now with 3 nodes and is there anything we can do to reduce those requirements?

Reply:
Can't you do something similar to other providers where we have KUBEVIRT_NUM_NODES environment variable. If that is 3, then the worker count should be 2, and if it is less than 3 set the worker count to 1 (I don't think we can have 0 workers and just a master can we?)

Reply:
in general, we have one master and one worker node, and we make master node schedulable for the workload, in case of other providers(except OKD) one, it less critical, because it does not  run such big number of system pods